src/GraphQL/Types/IImplementInterfaces.cs

```diff
@@ -3,10 +3,19 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// Provides properties for enumerating supported GraphQL interface types for an output graph type.
```

Comment:
`supported GraphQL interface types` - this is only one descendant - `IObjectGraphType`/`ObjectGraphType`.